plugins/toolbox/src/hooks.ts

```diff
@@ -24,3 +26,23 @@ export function useToolboxApi<T>(
     return await f(toolboxApi);
   }, deps);
 }
+
+export const useToolboxTranslation = () => {
+  const hookRef = useTranslationRef(toolboxTranslationRef);
+
+  const i18n_UNSAFE = useCallback(
```

Comment:
Cool but might not need this?

Reply:
the useCallback or the custom function?

I'll fix and reply the rest tomorrow.

Reply:
The custom function, but not sure if it can be overcome. If not, let's merge this after couple of fixes and think about that later

Reply:
or we could leave 
```ts
t(`tool.${tool.id}.name` as any, 'value fallback' as any)
```

everywhere that's being used. (right now I think it's being used in around 7 places(

Reply:
Let's keep the UNSAFE for now. Can you check the other comments?

Reply:
I did, I'll close the ones related to the UNSAFE and resolve the others.

Reply:
Also it might make sense to request a feature from backstage to add the defaultValue support :)

Reply:
and allow keys strings that aren't in the messages objects.

Reply:
All done.

anything more for now?